documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -389,7 +404,7 @@
             },
             annotations: {
               summary: 'More than half of the Prometheus instances within the same HA group are crashlooping.',
-              description: '{{ $value | humanizePercentage }} of Prometheus instances within the %(prometheusHAGroupName)s HA group have restarted at least 5 times in the last 30m.' % $._config,
+              description: '{{ $value | humanizePercentage }} of Prometheus instances within the %(prometheusHAGroupName)s HA group have restarted at least 5 times or their lockfile have been recreated 2 times (signaling an unclean exit) in the last 30m. ' % $._config,
```

Comment:
have → has

But beyond grammar, I think that's not quite what the alert is expressing. If there has been at least one start from an unclean storage _and_ there have been 2 or more restarts in the last 30m, the alert will fire.

The question is if that's the behavior we want. We should probably look at the last start, i.e. just look at the current value of `prometheus_tsdb_clean_start`. The last restart could have been with a fix of the problem (new configuration or a new binary), so if the last restart happened cleanly, we can arguable cease to alert immediately.

Another approach would be to check out how often `prometheus_tsdb_clean_start` was 0, so more `avg_over_time` than `min_over_time`.

Reply:
Makes a lot of sense. I switched it to use the last value